core/templates/dev/head/domain/question/EditableQuestionBackendApiService.ts

```diff
@@ -31,7 +31,7 @@ oppia.constant(
   'EDITABLE_QUESTION_DATA_URL_TEMPLATE',
   '/question_editor_handler/data/<question_id>');
 oppia.constant(
-  'QUESTION_CREATION_URL', '/question_editor_handler/create_new/<skill_id>');
+  'QUESTION_CREATION_URL', '/question_editor_handler/create_new/<skill_ids>');
```

Comment:
Should <skill_ids> be <comma_separated_skill_ids>?

Reply:
Done